contrib/win32/win32compat/win32_usertoken_utils.c

```diff
@@ -617,12 +618,32 @@ HANDLE generate_sshd_virtual_token()
 	if (AddSidMappingToLsa(&domain, &account, sid_user) != 0)
 		goto cleanup;
 
+	/* assign service logon privilege to virtual account */
+	{
+		LSA_OBJECT_ATTRIBUTES ObjectAttributes;
+		UNICODE_STRING svcLogonRight;
+		NTSTATUS lsa_ret;
+
+		ZeroMemory(&ObjectAttributes, sizeof(ObjectAttributes));
+		if ((lsa_ret = LsaOpenPolicy(NULL, &ObjectAttributes,

```

Comment:
Due to LsaOpenPolicy and LsaAddAccountRights, looks like this may suffer the same OneCore linking issue that I with issue [#1154](https://github.com/PowerShell/Win32-OpenSSH/issues/1154).

Reply:
Followed up. Looks like these APIs should be available on OneCore. We are having issues with OneCore building due to bugs in SDK. @bingbing8 is following up on working around these.